### PR TITLE
添加了“需要审核”的开关

### PR DESCRIPTION
**Pull request by OnceCemic**
添加了“需要审核”的开关，开启后普通用户将无法公开博客和发送评论

### CODE REVIEW

File: blog/views.py [hunk 1]
```diff
@@ -137,14 +140,17 @@ def form_valid(self, form):
...
+        instance.visible = False
       instance.save()
       instance.revisions.create(title=instance.title, text=instance.text, author=self.request.user)
     return redirect(reverse('blog:detail', kwargs=self.kwargs))
 
 
 class BlogAddComment(LoginRequiredMixin, View):
   def post(self, request, pk):
-    if 'text' in request.POST and request.POST['text']:
+    if 'text' in request.POST and request.POST['text'] \
```
Comment (ultmaster) [inline blog/views.py:152]: Can we hide the textarea on the frontend to prohibit AddComment when `review_requested`?

File: utils/site_settings.py [hunk 2]
```diff
@@ -41,6 +43,10 @@ def is_site_closed(request):
   return False
 
 
+def review_requested():
+  return literal_eval(site_settings_get("REVIEW_REQUESTED", default=False, use_cache=True))
```
Comment (ultmaster) [inline utils/site_settings.py:47]: Why `literal_eval`?

Comment (OnceCemic) [reply utils/site_settings.py:47]: site_settings_get returns a string, so eval is needed to convert it to bool; however, eval is somewhat unsafe, so literal_eval is used.

File: blog/views.py (hunk 1 above)
Comment (OnceCemic) [reply blog/views.py:153]: Fixed.

**Review by ultmaster — APPROVED** [review]

**Pull request merged**